### PR TITLE
Change the output of import commands to return arrays

### LINKED ISSUE #167: --output json should always create a single object
`kp import --output json` will create multiple json objects when this should only create one json object for processing

### PR DESCRIPTION
**Pull request by xtreme-shane-lattanzio**
Resolves #167

### CODE REVIEW

File: pkg/k8s/resource_printer.go [hunk 1]
```diff
@@ -61,19 +68,38 @@ func (y *YAMLObjectPrinter) PrintObject(obj runtime.Object, w io.Writer) error {
...
+
+	if len(objs) > 1 {
+		base.WriteRune('[')
 	}
 
-	var buf bytes.Buffer
-	err = json.Indent(&buf, data, "", "    ")
-	if err != nil {
-		return err
+	for _, obj := range objs {
```
Comment (chenbh) [inline pkg/k8s/resource_printer.go:78]: Is this change needed? `json.Marshal` should automatically add the start and ending brackets when passed in a slice.

File: pkg/commands/import/import_test.go [hunk 2]
```diff
@@ -786,122 +786,117 @@ Importing ClusterBuilder 'default'...
...
+  data:
+    image: default-registry.io/default-repo@sha256:lifecycle-image-digest
+  kind: ConfigMap
+  metadata:
+    annotations:
+      kpack.io/import-timestamp: "2006-01-02T15:04:05Z"
+    creationTimestamp: null
+    name: lifecycle-image
+    namespace: kpack
+- apiVersion: kpack.io/v1alpha2
```
Comment (chenbh) [inline pkg/commands/import/import_test.go:799]: For yaml we should keep the old format of it outputting multiple documents. It's a common practice to pipe the output of one command into another like `kp import descriptor.yaml --output yaml | kubectl apply -f -`, which would fail if it only outputs a single document with an array instead of multiple documents

**Review by chenbh — CHANGES_REQUESTED** [review]

File: pkg/k8s/resource_printer.go (hunk 1 above)
Comment (xtreme-shane-lattanzio) [reply pkg/k8s/resource_printer.go:78]: Each object is json but the whole thing is not so if you try to marshal it, it fails

**Review by chenbh — APPROVED** [review]
lgtm
